### PR TITLE
Added Italian translation

### PR DESCRIPTION
**Pull request by flauta**

### CODE REVIEW

**Review by dessalines — CHANGES_REQUESTED** [review]
Thx! Check the CI tho, lots of unicode escape errors.

**Review by flauta — COMMENTED** [review]
corrected non-escaped symbols

File: app/src/main/res/values-it/strings.xml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    <string name="source_code">Codice sorgente</string>
+    <string name="source_code_subtitle">Thumb-Key è un software libero e open source, sotto licenza GNU Affero General Public License v3.0</string>
+    <string name="look_and_feel">Aspetto</string>
+    <string name="behavior">Comportamento</string>
+    <string name="layouts">Schemi tastiera</string>
+    <string name="theme">Tema</string>
+    <string name="theme_color">Colore del tema</string>
+    <string name="position">Posizione</string>
+    <string name="vibrate_on_tap">Vibrazione al tocco</string>
+    <string name="vibrate_warning">È necessario attivare 'Vibra al tocco' nelle impostazioni del telefono</string>
```
Comment (flauta) [inline app/src/main/res/values-it/strings.xml:41]: <string name="vibrate_warning">È necessario attivare \'Vibra al tocco\' nelle impostazioni del telefono</string>

Comment (flauta) [inline app/src/main/res/values-it/strings.xml:41]: <string name="vibrate_warning">È necessario attivare \'Vibra al tocco\' nelle impostazioni del telefono</string>

File: app/src/main/res/values-it/strings.xml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    <string name="reset_to_defaults_msg">Confermare il ripristino delle impostazioni ?</string>
+    <string name="reset_to_defaults_confirm">Ripristina</string>
+    <string name="cancel">Annulla</string>
+    <string name="test_out_thumbkey">Prova Thumb-Key</string>
+    <string name="setup">Impostazioni</string>
+    <string name="user_guide">Guida d\'uso</string>
+    <string name="setup_thumbkey">Preparazione di Thumb-Key</string>
+    <string name="enable_thumbkey">Attivare Thumb-Key</string>
+    <string name="select_thumbkey">Selezionare Thumb-Key</string>
+    <string name="finish_setup">Concludere l'impostazione</string>
```
Comment (flauta) [inline app/src/main/res/values-it/strings.xml:68]: <string name="finish_setup">Concludere l\'impostazione</string>

**Review by flauta — COMMENTED** [review]
some non-escaped symbols

**Comment by flauta** [comment]
I read the help on the reviews: can you merge the new strings from the comments as they say?

**Comment by dessalines** [comment]
You need to escape things like single quotes.

**Comment by flauta** [comment]
> You need to escape things like single quotes.

Yes I did it, but in the comments the escape gets striped away again. 
I'd better close this PR and open another one.

**Pull request closed without merging**